ignite/contrib/engines/common.py

```diff
@@ -623,7 +626,7 @@ def gen_save_best_models_by_val_score(
         n_saved=n_saved,
         global_step_transform=global_step_transform,
         score_name=f"{tag}_{metric_name.lower()}",
-        score_function=Checkpoint.get_default_score_fn(metric_name),
+        score_function=Checkpoint.get_default_score_fn(metric_name, score_sign=score_sign),
```

Comment:
Out of the scope of this PR but let's make the code uniform to below `EarlyStopping` usage:
```suggestion
        score_function=get_default_score_fn(metric_name, score_sign=score_sign),
```